lib/node/runner/api/resource/frozen_account.go

```diff
@@ -0,0 +1,72 @@
+package resource
+
+import (
+	"strings"
+
+	"boscoin.io/sebak/lib/block"
+	"boscoin.io/sebak/lib/common"
+	"github.com/nvellon/hal"
+)
+
+type FrozenAccountState string
+
+const (
+	FrozenState    FrozenAccountState = "frozen"
+	MeltingState   FrozenAccountState = "melting"
+	UnfreezedState FrozenAccountState = "unfreezed"
+	RefundState    FrozenAccountState = "refund"
+)
+
+type FrozenAccount struct {
+	ba                           *block.BlockAccount
+	createdBlockheight           uint64
+	createdOpHash                string
+	createdSequenceId            uint64
+	initialAmount                common.Amount
+	freezingState                FrozenAccountState
+	unfreezingRequestBlockheight uint64
+	unfreezingRequestOpHash      string
+	paymentOpHash                string
+}
+
+func NewFrozenAccount(ba *block.BlockAccount, create_block_height uint64, create_opHash string, sequenceid uint64, amount common.Amount, state FrozenAccountState, unfreezing_block_height uint64, unfreezing_opHash string, payment_opHash string) *FrozenAccount {
```

Comment:
I have a question, Do you have any struct for this fields (sequenceid,create_opHash,state,unfreezing_XXX...) such as `FrozenAccount`?

Reply:
@anarcher I made a struct which have those fields. Thanks.